vectornav/params/vn100.yaml

```diff
@@ -8,12 +8,12 @@ serial_baud: 115200
 # Acceptable data rates in Hz: 1, 2, 4, 5, 10, 20, 25, 40, 50, 100, 200
 # Baud rate must be able to handle the data rate
 async_output_rate: 25
+min_async_output_rate: 12.5
+max_async_output_rate: 50
```

Comment:
@patapufate +1, but will it still be possible to tweak these from `system_testbed` tests? Tests need the ability to set (bad) rates.

Reply:
Checked.